src/libs/ReportUtils.js

```diff
@@ -1339,6 +1340,46 @@ function buildOptimisticCreatedReportAction(ownerEmail) {
     };
 }
 
+/**
+ * Returns the necessary reportAction onyx data to indicate that a task report has been edited
+ *
+ * @param {String} ownerEmail
+ * @returns {Object}
+ */
+
+function buildOptimisticEditedTaskReportAction(ownerEmail) {
+    return {
+        reportActionID: NumberUtils.rand64(),
+        actionName: CONST.REPORT.ACTIONS.TYPE.TASKEDITED,
+        pendingAction: CONST.RED_BRICK_ROAD_PENDING_ACTION.ADD,
+        actorAccountID: currentUserAccountID,
+        actorEmail: currentUserEmail,
```

Comment:
@thienlnam I narrowed it down and added this in, which I _believe_ is the fix for the `replace` error that we've been getting.